plutus-core/plutus-core/src/PlutusCore/Evaluation/Machine/ExBudgetingDefaults.hs

```diff
@@ -74,14 +75,19 @@ defaultCekCostModel = CostModel defaultCekMachineCosts defaultBuiltinCostModel
 defaultCostModelParams :: Maybe CostModelParams
 defaultCostModelParams = extractCostModelParams defaultCekCostModel
 
+defaultUnliftingMode :: UnliftingMode
+defaultUnliftingMode = UnliftingDeferred
+
 defaultCekParameters :: MachineParameters CekMachineCosts CekValue DefaultUni DefaultFun
-defaultCekParameters = mkMachineParameters defaultCekCostModel
+defaultCekParameters = mkMachineParameters defaultUnliftingMode defaultCekCostModel
 
 unitCekParameters :: MachineParameters CekMachineCosts CekValue DefaultUni DefaultFun
-unitCekParameters = mkMachineParameters (CostModel unitCekMachineCosts unitCostBuiltinCostModel)
+unitCekParameters =
+    mkMachineParameters defaultUnliftingMode $
+        CostModel unitCekMachineCosts unitCostBuiltinCostModel
 
 defaultBuiltinsRuntime :: HasConstantIn DefaultUni term => BuiltinsRuntime DefaultFun term
-defaultBuiltinsRuntime = toBuiltinsRuntime defaultBuiltinCostModel
+defaultBuiltinsRuntime = toBuiltinsRuntime UnliftingImmediate defaultBuiltinCostModel
```

Comment:
`defaultUnliftingMode`?

Reply:
No idea how that happened. Thanks for catching.